docs/docs-graphql/schema.graphqls

```diff
@@ -2454,13 +2454,13 @@ type GitRepository {
   withAuthHeader(
     """Secret used to populate the Authorization HTTP header"""
     header: SecretID!
-  ): GitRepository!
+  ): GitRepository! @deprecated(reason: "Use \"httpAuthHeader\" instead.")
```

Comment:
Maybe add "in the constructor" (or similar) to clarify?

Reply:
:tada: yup agreed! Thanks :pray: